rust/src/measurements/gumbel_max/mod.rs

```diff
@@ -20,22 +18,20 @@ use crate::traits::{
     DistanceConstant,
 };
 
-#[derive(PartialEq, Clone)]
+#[derive(PartialEq, Clone, Copy)]
+#[cfg_attr(feature = "polars", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "polars", serde(rename_all = "lowercase"))]
 pub enum Optimize {
     Min,
     Max,
 }
 
-impl Display for Optimize {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(
-            f,
-            "{}",
-            match *self {
-                Optimize::Min => "min",
-                Optimize::Max => "max",
-            }
-        )
+impl std::fmt::Debug for Optimize {
```

Comment:
Is it always better to define `Debug` rather than `Display`? Or is the change particular to what's going on in this PR?

Reply:
I've updated to the approach that Debug is as automatically derived by the macro (uppercase), and Display is lowercase.